src/test/skript/tests/syntaxes/conditions/CondMatches.sk

```diff
@@ -0,0 +1,3 @@
+test "matches condition":
```

Comment:
A few more tests would be nice... matching uuid? etc

Reply:
I'll add them